tests/src/test/scala/com/nvidia/spark/rapids/HashAggregatesSuite.scala

```diff
@@ -271,7 +271,7 @@ class HashAggregatesSuite extends SparkQueryCompareTestSuite {
       "avg literals dbl_max",
       longsFromCSVDf,
       maxFloatDiff = 0.0001,
-      conf = floatAggConf.set(RapidsConf.INCOMPATIBLE_OPS.key, "true")) {
+      conf = floatAggConf.clone.set(RapidsConf.INCOMPATIBLE_OPS.key, "true")) {
```

Comment:
we could make this a one line fix by turning val into `def floatAggConf`.  It will also make it less likely for a new test to re-introduce the bug.